src/R/Editor/Impl/Completions/Providers/ParameterNameCompletionProvider.cs

```diff
@@ -49,7 +49,10 @@ public class ParameterNameCompletionProvider : IRCompletionListProvider {
             // Collect parameter names from all signatures
             IEnumerable<KeyValuePair<string, IArgumentInfo>> arguments = new Dictionary<string, IArgumentInfo>();
             foreach (var signature in functionInfo.Signatures) {
-                var args = signature.Arguments.ToDictionary(x => x.Name);
+                var args = new Dictionary<string, IArgumentInfo>();
```

Comment:
It could've been `signature.Arguments.Where(x => !string.IsNullOrEmpty(x.Name)).ToDictionary(x => x.Name);`, but up to you